src/ert/analysis/_es_update.py

```diff
@@ -305,13 +305,15 @@ def _get_obs_and_measure_data(
     for obs_key, obs_active_list in selected_observations:
         observation = observations[obs_key]
         group = observation.attrs["response"]
+
```

Comment:
There seems to be missing something here?

Reply:
Its just an extra line